packages/components/src/index.ts

```diff
@@ -145,6 +145,8 @@ import {
     formatDateTime,
     getDateFormat,
     getDateTimeFormat,
+    isDateInPast,
+    isDateTimeInPast,
```

Comment:
I see the use of `isDateTimeInPast` in the `ExperiationDateColumnRenderer` but is `isDateInPast` used? (it's likely I just missed it)

Reply:
You are right isDateInPast is not used. I'll keep isDateInPast in utils but remove from export. The original preferred implementation is we don't care about timestamp portion of expiration date. We went with date and time for expiration date, but keeping the util in case we flip in the future.